src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -1935,6 +1935,12 @@ material.blue_alloy=Blue Alloy
 material.lapotron=Lapotron
 material.aluminium_sulfite=Aluminium Sulfite
 
+material.cyclohexane=Cyclohexane
+material.cyclohexanoxime=Cyclohexanoxime
```

Comment:
In accordance with my other comment on this compound: the unlocalized name should be `cyclohexanone_oxime` and the translation should be `Cyclohexanone Oxime`.